ProcessLib/LiquidFlow/LiquidFlowLocalAssembler-impl.h

```diff
@@ -50,6 +53,55 @@ void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
 
 template <typename ShapeFunction, typename IntegrationMethod,
           unsigned GlobalDim>
+void LiquidFlowLocalAssembler<ShapeFunction, IntegrationMethod, GlobalDim>::
+    coupling_assemble(double const t, std::vector<double> const& local_x,
+                      std::vector<double>& local_M_data,
+                      std::vector<double>& local_K_data,
+                      std::vector<double>& local_b_data,
+                      LocalCouplingTerm const& coupled_term)
+{
+    SpatialPosition pos;
+    pos.setElementID(_element.getID());
+    const int material_id = _material_properties.getMaterialID(pos);
+
+    const Eigen::MatrixXd& perm = _material_properties.getPermeability(
+        material_id, t, pos, _element.getDimension());
+    // Note: For Inclined 1D in 2D/3D or 2D element in 3D, the first item in
+    //  the assert must be changed to perm.rows() == _element->getDimension()
+    assert(perm.rows() == GlobalDim || perm.rows() == 1);
+
+    const double dt = coupled_term.dt;
+    for (auto const& coupled_process_map : coupled_term.coupled_processes)
+    {
+        if (coupled_process_map.first ==
```

Comment:
I would like to keep this in case there will be other coupled processes to be added on.